openbb_terminal/cryptocurrency/onchain/dune_model.py

```diff
@@ -0,0 +1,119 @@
+import pandas as pd
```

Comment:
hi, I recommend you use `dune-client` [from](https://pypi.org/project/dune-client/)

or at the very least, given you want to use this with pandas, use this:
https://github.com/cowprotocol/dune-client/blob/35e514c9f5330effc1b70dc01e2b8835c23845cc/dune_client/client.py#L107

you want to use the `CSV` API and load that directly into `pandas.read_csv()`

that way you can handle larger responses >500MB with better latency, performance and memory utilization.